suite-common/message-system/src/__tests__/experimentUtils.test.ts

```diff
@@ -61,13 +82,9 @@ describe('testing experiment utils', () => {
         const shareA = groupACount / sampleSize;
         const shareB = groupBCount / sampleSize;
 
-        expect(shareA).toBeGreaterThanOrEqual(
-            experimentTest.groups[0].percentage / 100 - deviation,
-        );
-        expect(shareA).toBeLessThanOrEqual(experimentTest.groups[0].percentage / 100 + deviation);
-        expect(shareB).toBeGreaterThanOrEqual(
-            experimentTest.groups[1].percentage / 100 - deviation,
-        );
-        expect(shareB).toBeLessThanOrEqual(experimentTest.groups[1].percentage / 100 + deviation);
+        expect(shareA).toBeGreaterThanOrEqual(experimentTest.groups[0].percentage / 100);
+        expect(shareA).toBeLessThanOrEqual(experimentTest.groups[0].percentage / 100);
+        expect(shareB).toBeGreaterThanOrEqual(experimentTest.groups[1].percentage / 100);
+        expect(shareB).toBeLessThanOrEqual(experimentTest.groups[1].percentage / 100);
```

Comment:
Without the deviation, `>= && <=` can be simplified as `==`, right?
```suggestion
        expect(shareA).toEqual(experimentTest.groups[0].percentage / 100);
        expect(shareB).toEqual(experimentTest.groups[1].percentage / 100);
```